@navikt/ds-react/layouts/product-page/Layout.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from "react";
+import cl from "classnames";
+import { ForwardRefExoticComponent, HTMLAttributes } from "react";
+import { Children, forwardRef } from "react";
+import { default as Section, SectionProps } from "./Section";
+import { ContentContainer, Heading } from "../../";
+import { whiteDecoratorUtils } from "../common/utils";
+import "@navikt/ds-css/layouts/index.css";
+
+export interface LayoutWithSubComponents
+  extends ForwardRefExoticComponent<LayoutProps> {
+  Section: ForwardRefExoticComponent<SectionProps>;
+}
+
+export interface LayoutProps extends HTMLAttributes<HTMLElement> {
+  title: string;
+  children?: React.ReactNode;
+}
+
+const Layout = forwardRef<HTMLDivElement, LayoutProps>(
+  ({ children, title, className, ...rest }, ref) => {
+    const columns = Children.count(children);
+    const classNames = cl(
+      "navds-layout__container",
+      `navds-layout__container--${columns}-columns`,
+      className
+    );
+
+    useEffect(() => {
+      whiteDecoratorUtils(true);
+      return () => {
+        whiteDecoratorUtils(false);
+      };
+    }, []);
+
+    return (
+      <div ref={ref}>
+        <div className={"navds-layout__container--white"}>
+          <ContentContainer>
```

Comment:
Har vi gått vekk fra at ContentContainer er noe som wrapper layout?

Reply:
Om tittelfeltet vårt skal ha hvit bakgrunnsfarge iht ønskene fra designerne, så må vi ha en egen seksjon med egen styling:

<img width="1428" alt="Screenshot 2021-03-10 at 09 18 24" src="https://user-images.githubusercontent.com/6075402/110598062-9f66a280-8181-11eb-9109-8e77458ea9f3.png">

Ergo vil man ha to contentContainere over hverandre for å løse problemet, men jeg synes det funker ganske fint nå :) 

Så da blir contentContainer en del av komponenten / layouten som kan benyttes slik:
```
<ProductPageLayout title={title}>
    <ProductPageLayout.Section left sticky>
      <Left />
    </ProductPageLayout.Section>
    <ProductPageLayout.Section>
      <Main />
    </ProductPageLayout.Section>
    <ProductPageLayout.Section right sticky>
      <Right />
    </ProductPageLayout.Section>
  </ProductPageLayout>
  ```

Reply:
Det gir mening. 👍